polaris.shopify.com/src/components/Markdown/components/DoDont/styles.module.scss

```diff
@@ -18,64 +19,10 @@
   }
 }
 
-.DoDontPart {
-  font-size: var(--font-size-300);
-  position: relative;
-  flex: 1;
-  border-radius: var(--border-radius-400);
-  background-color: var(--surface);
-
-  h4:first-child {
-    display: inline-block;
-    background-color: var(--fill-success);
-    color: var(--text-success-onfill);
-    padding: var(--p-space-050) var(--p-space-200) var(--p-space-050)
-      var(--p-space-500);
-    border-radius: var(--p-space-200);
-    font-size: var(--font-size-100);
-    font-weight: var(--font-weight-600);
-    line-height: 1rem;
-    position: relative;
-  }
-
-  h4:first-child:after {
-    content: '';
-    display: block;
-    position: absolute;
-    left: 0;
-    top: 0;
-    height: 20px;
-    width: 20px;
-    background-image: url(/images/icon-do.svg);
-    background-repeat: no-repeat;
-    background-position: left center;
-    background-size: cover;
-    z-index: 1;
-  }
-
-  img {
-    width: auto;
-    max-width: 100%;
-  }
-
-  h3 {
-    font-size: var(--font-size-300);
-    color: var(--text);
-  }
-
-  p {
-    margin: var(--p-space-200) 0 0 0;
-    font: var(--font-body-md);
-  }
+.Do {
```

Comment:
This is so that `Do` and `Dont` cards rendered in a `DoDont` component, are always placed in the correct  `grid column`. 

(Do cards should render on the left and Dont cards on the right)